components/patient-data-sharing/push-ui/src/main/resources/PhenoTips/PushPatient.xml

```diff
@@ -218,40 +207,40 @@
       this._container.insert(generateSection('server-selection', serverTitle, serverDesc))
                      .insert(generateSection('user-selection', "$services.localization.render('phenotips.patientSharing.push.dialog.authentication.label')","$services.localization.render('phenotips.patientSharing.push.dialog.authentication.hint')"))
                      .insert(generateSection('group-selection',"$services.localization.render('phenotips.patientSharing.push.dialog.remoteGroup.label')","$services.localization.render('phenotips.patientSharing.push.dialog.remoteGroup.hint')"));
-      if (this._consentsModulePresent) {
-        this._container.insert(generateSection('consents-section',
-          "$services.localization.render('phenotips.patientSharing.push.dialog.consents.label')",
-          "$services.localization.render('phenotips.patientSharing.push.dialog.consents.hint')"));
-      }
-      if (!this._pushManyPatients) {
-          this._container.insert(generateSection('fields-selection',"$services.localization.render('phenotips.patientSharing.push.dialog.preview.label')","$services.localization.render('phenotips.patientSharing.push.dialog.preview.hint')"));
-      }
+
+      this._container.insert(generateSection('consents-section',
+        "$services.localization.render('phenotips.patientSharing.push.dialog.consents.label')",
+        "$services.localization.render('phenotips.patientSharing.push.dialog.consents.defaulthint')"));
+
+      // TODO: a sectionto preview patient data being pushd. It used to be easy, but
+      //       now that we push a lot of different fields the preview is disabled until
+      //       we come up with an easy way to display all of the data
+      // if (!this._pushManyPatients) {
+      //   this._container.insert(generateSection('fields-selection',"$services.localization.render('phenotips.patientSharing.push.dialog.preview.label')","$services.localization.render('phenotips.patientSharing.push.dialog.preview.hint')"));
+      //   this._fieldsManager = this._container.down('#fields-selection .section-contents');
+      // } else {
```

Comment:
might have commented on this before, but commented/out code shouldn't be commited. We have the git history to go look for it :)